flink-connectors/flink-connector-gcp-pubsub/src/main/java/org/apache/flink/streaming/connectors/gcp/pubsub/table/PubSubConnectorConfigOptions.java

```diff
@@ -0,0 +1,27 @@
+package org.apache.flink.streaming.connectors.gcp.pubsub.table;
+
+import org.apache.flink.annotation.PublicEvolving;
+import org.apache.flink.configuration.ConfigOption;
+import org.apache.flink.configuration.ConfigOptions;
+
+/**
+ * Options for PubSub tables supported by the {@code CREATE TABLE ... WITH ...} clause of the Flink
+ * SQL dialect and the Flink Table API.
+ */
+@PublicEvolving
+public class PubSubConnectorConfigOptions {
```

Comment:
Can we please rename this to `PubSubConnectorOptions`? We recently went through some changes and want to make this consistent.

Reply:
@brachi-wernick This comment is still open, isn't it?